README.md

```diff
@@ -43,7 +43,7 @@ Docker development environment on OS X, including installing Docker and
 Boot2Docker:
 
 ```sh
-curl -o /usr/local/bin/docker-osx-dev https://raw.githubusercontent.com/brikis98/docker-osx-dev/master/src/docker-osx-dev
+curl -o /usr/local/bin/docker-osx-dev https://raw.githubusercontent.com/devotox/docker-osx-dev/master/src/docker-osx-dev
```

Comment:
Thanks for the PR. In addition to what @brikis98 already said, please also keep the official path in the docu.